### PR TITLE
Open at any tapemark

### PR DESCRIPTION
**Pull request by ErlendHaa**

### CODE REVIEW

File: test/tapeimage.cpp [hunk 1]
```diff
@@ -433,6 +426,81 @@ TEST_CASE(
...
+    }
+
+    SECTION( "Seek past eof" ) {
+         const auto err = lfp_seek(tif, 10);
+         CHECK(err == LFP_OK);
+
+         std::int64_t tell = -1;
+         lfp_tell(tif, &tell);
+         CHECK(tell == 10);
+    }
```
Comment (achaikou) [inline test/tapeimage.cpp]: might be useful to do "peek and tell" to verify your claim on underlying file

> the tell of eof + the remaining requested bytes

so 34 in your case.

File: test/tapeimage.cpp [hunk 2]
```diff
@@ -885,20 +885,34 @@ TEST_CASE(
...
     SECTION( "Seek with index" ) {
         err = lfp_seek(tif, 2);
-        CHECK(err == 0);
+        CHECK(err == LFP_OK);
 
         std::int64_t tell;
         lfp_tell(tif, &tell);
         CHECK(tell == 2);
     }
+    SECTION( "Read from already index records" ) {
```
Comment (achaikou) [inline test/tapeimage.cpp]: index(ed)

File: include/lfp/tapeimage.h [hunk 3]
```diff
@@ -21,6 +21,11 @@ extern "C" {
  * present. `lfp_seek()` and `lfp_tell()` consider offsets as if the file had
  * no tape markers.
  *
+ * The protocol can be opened at any tapemark within a file. Tells will start
+ * at that tapemark, meaning any previous records are unreachable by the
+ * protocol. Note that its not possible to open the protocol in the middle of
```
Comment (achaikou) [inline include/lfp/tapeimage.h]: it(')s = it is

Comment (ErlendHaa) [reply include/lfp/tapeimage.h]: fixed

File: test/tapeimage.cpp (hunk 2 above)
Comment (ErlendHaa) [reply test/tapeimage.cpp]: fixed

**Pull request merged**